docs/docs-beta/package.json

```diff
@@ -15,7 +15,8 @@
     "ts": "tsc -p . --noEmit",
     "vale": "vale ./docs --ext=.md,.mdx",
     "lint": "eslint . --ext=.tsx,.ts,.js,.md,.mdx --fix",
-    "lint-and-vale": "yarn run lint && yarn run vale"
+    "lint-and-vale": "yarn run lint && yarn run vale",
+    "sync-api-docs": "/bin/sh scripts/vercel-sync-api-docs.sh"
```

Comment:
New directive for yarn to call our vercel api doc build script.

Reply:
please make sure you update the docs-beta/README as well!

Reply:
Good call, updated README here: https://github.com/dagster-io/dagster/pull/24362/commits/5a741bb306beeb573416b7c962c5b7ecb766a962